projects/go-lib/src/lib/components/go-table/go-table.component.spec.ts

```diff
@@ -399,8 +387,12 @@ describe('GoTableComponent', () => {
     });
 
     it('should set selectAllIndeterminate to false if selectAll toggled off', () => {
+      component.ngOnInit();
+
       component.selectAllControl.setValue(false);
 
+      component['selectionSubscriptions'].unsubscribe();
```

Comment:
Does it not work if we call `component.ngOnDestroy()` here? I believe throughout goponents we're using `takeUntil` to clean up after subscriptions instead of using the unsubscribe method.

Reply:
Yeahhhh, you're right. This was like my 20th attempt to fix the issue so I forgot about cleaning this up lol. I switched everything to use the `ngOnDestroy` method instead and was able to remove the subscription.